default/python/AI/InvasionAI.py

```diff
@@ -327,14 +327,13 @@ def _get_path_from_capital(planet: "fo.planet") -> Tuple[Sequence[SystemId], int
     """
     universe = fo.getUniverse()
     capitol_id = PlanetUtilsAI.get_capital()
-    if capitol_id:
+    if capitol_id and capitol_id != INVALID_ID:
```

Comment:
`PlanetUtilsAI.get_capital()` will return id or INVALID_ID, so we could drop` capitol_id and` part